### PR TITLE
Catalan-Spain translation

### PR DESCRIPTION
**Pull request by Ordained-SubGenii**
translation for Catalan language and Spain region (Catalunya)

### CODE REVIEW

**Comment by guyluz11** [comment]
It looks like you have the en-us.json translation file.

And you have some weird translation there on the wa
word area.

Screenshot_20221019-183207_Firefox


And please also delete all excess spacing in your translations

**Comment by Ordained-SubGenii** [comment]
I updated the catalan translation file. I accidentally had pushed the wrong file and english file the first time, apologies. let me know if any further adjustments are needed. thanks,

**Comment by guyluz11** [comment]
It looks like you have still removed English translation file.

And fix `"Room_name_is_required:"` to be `"Room_name_is_required":`

**Comment by guyluz11** [comment]
Screenshot_20221024_163153
 Please do not remove English translation

File: assets/translations/en-US.json [hunk 1]
```diff
@@ -110,3 +110,4 @@
   "Waiting_for_user_to_get_connected_to_WiFi": "Waiting for user to get connected to WiFi",
   "Can't_add_unsupported_device": "Can't add unsupported device"
 }
+
```
Comment (guyluz11) [inline assets/translations/en-US.json:113]: Please don't change en-US translation

**Pull request closed without merging**